charts/telescope-backend/Chart.yaml

```diff
@@ -6,4 +6,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.2
+version: 0.3.4
```

Comment:
Is there a reason we did not go to `0.3.3`?

Reply:
My bad, changed to 0.3.3